### PR TITLE
 Test deployment of single node hana using azure pipelines

### PR DESCRIPTION
**Pull request by persiaAziz**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,4 +23,16 @@ Execution-focused
...
+Coding Guidelines
+-----------------
+* Test:
+  * Azure pipeline is integrated with this repo that invokes build checks on the submitted pull requests. The pipeline runs the tests on the merge branch. The pull request is required to pass the Azure pipelines test before it can be merged.
+* Branch Policy:
+  * The contributor needs to create a branch in the main repo so that the integrated Azure pipelines can test the submitted pull request.
+  * The Azure pipeline tests actually deploy resources based on the source branch to validate and verify the submitted pull request. Please note that the deployed resource group and the domain name are named after the source branch to allow exclusive test deployments for multiple pull requests and branches. So the branch name can only contain small case alphanumeric characters and hyphens.
+  A recommended format of the branch name can be:
+  ```sh
+  <Contributors github handle>-<feature>-<name>
```
Comment (pabowers) [inline CONTRIBUTING.md]: nit: Contributor's github handle

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -23,4 +23,16 @@ Execution-focused
 -----------------
 * We don't just work on some grand plan that may or never be completely executed;
   * instead, we start building out the solution and iterate towards a grand plan.
+
+Coding Guidelines
+-----------------
+* Test:
+  * Azure pipeline is integrated with this repo that invokes build checks on the submitted pull requests. The pipeline runs the tests on the merge branch. The pull request is required to pass the Azure pipelines test before it can be merged.
```
Comment (tniek) [inline CONTRIBUTING.md]: nit: re-write, add link: "This repository integrates with [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/), which invokes build checks on the submitted pull requests."

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -23,4 +23,16 @@ Execution-focused
 -----------------
 * We don't just work on some grand plan that may or never be completely executed;
   * instead, we start building out the solution and iterate towards a grand plan.
+
+Coding Guidelines
+-----------------
+* Test:
```
Comment (tniek) [inline CONTRIBUTING.md]: nit: "Test" (no colon)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -23,4 +23,16 @@ Execution-focused
...
+  * Azure pipeline is integrated with this repo that invokes build checks on the submitted pull requests. The pipeline runs the tests on the merge branch. The pull request is required to pass the Azure pipelines test before it can be merged.
+* Branch Policy:
```
Comment (tniek) [inline CONTRIBUTING.md]: nit: "Branch Policy" (no colon)

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -23,4 +23,16 @@ Execution-focused
...
+  * The contributor needs to create a branch in the main repo so that the integrated Azure pipelines can test the submitted pull request.
```
Comment (tniek) [inline CONTRIBUTING.md]: nit: "Azure Pipelines"

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -23,4 +23,16 @@ Execution-focused
...
+  * The Azure pipeline tests actually deploy resources based on the source branch to validate and verify the submitted pull request. Please note that the deployed resource group and the domain name are named after the source branch to allow exclusive test deployments for multiple pull requests and branches. So the branch name can only contain small case alphanumeric characters and hyphens.
```
Comment (tniek) [inline CONTRIBUTING.md]: Highlight/bold the last sentence: "**So, the branch name can only contain small case alphanumeric characters and hyphens**."

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -23,4 +23,16 @@ Execution-focused
...
+  A recommended format of the branch name can be:
+  ```sh
```
Comment (tniek) [inline CONTRIBUTING.md]: Why sh? (typo?)

File: deploy/vm/test/terraform.tfvars [hunk 8]
```diff
@@ -1,11 +1,8 @@
-az_region = "westus"
+az_region = "eastus"
```
Comment (tniek) [inline deploy/vm/test/terraform.tfvars]: Switch region for SKU availability?

File: CONTRIBUTING.md (hunk 7 above)
Comment (persiaAziz) [reply CONTRIBUTING.md]: To have a code style format for the following line:
<Contributors github handle>-<feature>-<name>

File: deploy/vm/test/terraform.tfvars (hunk 8 above)
Comment (persiaAziz) [reply deploy/vm/test/terraform.tfvars]: yes. The subscription has quotas for D series in eastus only

**Pull request merged**